Objects/Converters/ConverterRevit/ConverterRevitTests/ConverterRevitTestsShared/SpeckleConversionTest.cs

```diff
@@ -186,16 +187,14 @@ await APIContext
         foreach (var el in spkElems)
         {
           object res = null;
-#pragma warning disable CA1031 // Do not catch general exception types
           try
           {
             res = converter.ConvertToNative(el);
           }
-          catch (Exception e)
+          catch (Exception ex) when (!ex.IsFatal())
           {
-            converter.Report.LogConversionError(new Exception(e.Message, e));
+            converter.Report.LogConversionError(new Exception(ex.Message, ex));
```

Comment:
If we're not adding any new information to the exception, we might as well just log that instead, as this just duplicates the information in the message.

LogConversionError will take care of the rest.

Reply:
Good point